history-of-the-quakers-v2/original/08-book-3-chapter-14.adoc

```diff
@@ -25,26 +31,29 @@ pestilence, or any symptom of divine displeasure, were calamities of equal magni
 with the existence of non-conformity, they proceeded to enact a fresh penal law,
 commonly known by the name of The Oxford five-mile act,
 which received the royal assent October the 31st, 1665.
-It was entitled, An aft to restrain non-conformists from inhabiting corporations;
-the preamble to which sets forth,
+It was entitled, An Act to Restrain Non-conformists from Inhabiting Corporations;
+the reamble to which sets forth,
 
-"`that divers Parsons and others in holy orders,
+[.embedded-content-document.legal]
+--
+
+that divers Parsons and others in holy orders,
```

Comment:
It's strange how he starts in the middle of sentence here, and even though what you did is exactly correct, it looks like an error. Can you add an ellipses on line 40? There's a special code to insert those, it's `&hellip;`, so the line would look like this (no space after the ellipses):

```ASCIIDOC
&hellip;that divers Parsons and others in holy orders,
```